tools/build_script_generator/scons/resources/SConscript.in

```diff
@@ -128,6 +129,11 @@ env["CONFIG_AVRDUDE_OPTIONS"] = "{{ avrdude_options }}"
 env["CONFIG_AVRDUDE_BAUDRATE"] = "{{ avrdude_baudrate }}"
 %% endif
 %% elif core.startswith("cortex-m")
+%% if partname == "rp2040"
+pico_openocd_path = environ.get('PICO_OPENOCD_PATH')
+if pico_openocd_path:
+  env.Append(MODM_OPENOCD_BINARY = pico_openocd_path)
```

Comment:
Why not use `MODM_OPENOCD_BINARY` in environ too and enable it for all targets? Could be useful when dealing with multiple custom openocd binaries for very new STM32 too.